app/assets/stylesheets/application.scss

```diff
@@ -15,3 +15,25 @@
  *= require foundation_and_overrides
 
  */
+@import url(https://fonts.googleapis.com/css?family=Alfa+Slab+One);
+
+.home-page {
+  width: 100%;
+  height: 100%;
+  background: url(/assets/panama_lineup.jpg);
+  background-size: cover;
+}
+
+.welcome {
+  border: 1px solid white;
+  background-color: rgba(0,0,0,.3);
```

Comment:
`.3` should be written with a leading zero as `0.3`<br>Commas in function arguments should be followed by a single space